<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vota</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #eee;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="seu_voto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>SEU VOTO PARA</string>
    </property>
   </widget>
   <widget class="QLabel" name="cargo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Presidente</string>
    </property>
   </widget>
   <widget class="QLabel" name="cap_numero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Número:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dig1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dig2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cap_nome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nome">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cap_partido">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Partido:</string>
    </property>
   </widget>
   <widget class="QLabel" name="partido">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="simulacao">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #aaa;</string>
    </property>
    <property name="text">
     <string>Simulação</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rodape_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>366</y>
      <width>611</width>
      <height>3</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rodape_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Aperte a tecla:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rodape_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>VERDE para CONFIRMAR este voto</string>
    </property>
   </widget>
   <widget class="QLabel" name="rodape_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>LARANJA para REINICIAR este voto</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto_mold">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black; background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="foto_vice_mold">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>213</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black; background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="foto_cargo">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>185</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Presidente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="foto_vice_cargo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Vice-Presidente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="foto">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>5</y>
      <width>160</width>
      <height>175</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="foto_vice">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>215</y>
      <width>110</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numero_errado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>NÚMERO ERRADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="voto_nulo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>VOTO NULO</string>
    </property>
   </widget>
   <widget class="QLabel" name="fim">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>FIM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
